test/output/callbacks/callbacks_test.py

```diff
@@ -90,6 +95,74 @@ def test_test_run_from_test_record(self, user_mock):
             b'This is a text file attachment.\n',
             parameter.value_binary,
         )
+        break
+    else:
+      raise AssertionError('No attachment named %s' % attachment_name)
+
+
+class TestMfgEventOutput(test.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    # Create input record.
+    result = util.NonLocalResult()
+    def _save_result(test_record):
+      result.result = test_record
+    cls._test = htf.Test(
+        all_the_things.hello_world,
+        all_the_things.dimensions,
+        all_the_things.attachments,
+        # We intentionally call dimensions and attachments phases twice so we
+        # can check functionality for non-unique measurement and attachment
+        # names.
+        all_the_things.hello_world,
+        all_the_things.attachments,
+    )
+    cls._test.add_output_callbacks(_save_result)
+    cls._test.make_uid = lambda: 'UNITTEST:MOCK:UID'
+
+  @test.patch_plugs(user_mock='openhtf.plugs.user_input.UserInput')
+  def test_mfg_event_from_test_record(self, user_mock):
+    user_mock.prompt.return_value = 'SomeWidget'
+    record = yield self._test
+
+    mfg_event = mfg_event_converter.mfg_event_from_test_record(record)
+
+    callback = console_summary.ConsoleSummary()
+    callback(record)
```

Comment:
lines 131 and 132 can be merged into one:
console_summary.ConsoleSummary(record)

Reply:
Technically, it would need an extra ():
`console_summary.ConsoleSummary()(record)`

I find this a bit confusing though because it obscures the fact that console_summary.ConsoleSummary() is a callable, so I think it's worth the extra line and variable.